Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 147 style the contact page

## Related ticket:
[SCRUM-147](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-147?atlOrigin=eyJpIjoiNTZjZTIxZDdjYTcxNDkwN2I3NjMxZThjZDk1MzE5ZmQiLCJwIjoiaiJ9)

## In this PR:
- Updated db (1.8.gz), content for contact info section
- Display content in the contact info section
- Layout adjustments and styling

## Testing instruction
- Checkout branch
- Import the latest db and config
- Check the Contact page
- Test if Contact form is working


[SCRUM-147]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
